RELEASE.md

```diff
@@ -1,6 +1,22 @@
 # Release History
 *****************
 
+## Release ONDEWO NLU Python Client 2.4.0
+
+### New features
+* [OND211-1836] Support intent tags with filtering
+* [OND211-1850] API Update. Allow sorting and searching of entity values on listing
+* [OND211-1850] API Update. Enable Entity Type View on all Entity Type CRUD operations
+
+### Improvements
+* [OND211-1911] Updated ondewo-logging version
+* Updated requirements compatibility
```

Comment:
It would be nice if every release note is associated to its task.
If there is no task, just associate it to the closest existing task. :)